.travis.yml

```diff
@@ -20,7 +18,5 @@ addons:
     packages:
     - libcurl4-openssl-dev
 
-cache: bundler # caching bundler gem packages will speed up build
```

Comment:
We *do* want to cache!

Reply:
RIGHT. But also, did you disable caching in the ci branch PRs for rockarch.org and Bits & Bytes?

Reply:
RIGHT